.github/actions/pipenv/requirements.txt

```diff
@@ -0,0 +1 @@
+pipenv==2023.6.12
```

Comment:
there is also a dependabot configuration that needs to be moved in order to keep this updated